### DISCUSSION TITLE
I want web Vue to use rabbitmq. Which ports should be opened?

### DISCUSSION THREAD

**Discussion by bbhxwl** [rabbitmq/rabbitmq-server#4978] [Other]
I am currently running with the following code.

docker run -itd  --restart=always --name rabbitmq -p 5672:5672 -p 15672:15672 rabbitmq:management

**Comment by michaelklishin** [comment]
The list of ports used by RabbitMQ [is documented](https://www.rabbitmq.com/networking.html#ports). You probably want to consider Web STOMP or Web MQTT in a Web UI client.

**Reply by bbhxwl** [reply]
> The list of ports used by RabbitMQ [is documented](https://www.rabbitmq.com/networking.html#ports). You probably want to consider Web STOMP or Web MQTT in a Web UI client.

Thank you. I will enable plug-ins after I enter docker, but I will not enable all plug-ins when docker runs.

**Reply by michaelklishin** [reply]
[How to Preconfigure Plugins at Deployment Time](https://rabbitmq.com/plugins.html#enabled-plugins-file). This is not really a place to get help with Docker practices.
